src/test/java/edu/hm/hafner/analysis/parser/PylintParserTest.java

```diff
@@ -219,6 +219,24 @@ void shouldParseReportWithoutSymbol() {
                     .hasFileName("trunk/src/python/tv.py")
                     .hasCategory("W0102")
                     .hasSeverity(Severity.WARNING_NORMAL);
+            softly.assertThat(iterator.next())
+                    .hasLineStart(1)
+                    .hasLineEnd(1)
+                    .hasMessage("Unused import os (unused-import)")
+                    .hasFileName("trunk/src/python_package/module_name.py")
+                    .hasCategory("W0611")
+                    .hasSeverity(Severity.WARNING_NORMAL)
+                    .hasModuleName("python_package.module_name")
+                    .hasPackageName("python_package");
+            softly.assertThat(iterator.next())
+                    .hasLineStart(1)
+                    .hasLineEnd(1)
+                    .hasMessage("Unused import os (unused-import)")
+                    .hasFileName("trunk/src/module_name_no_package.py")
+                    .hasCategory("W0611")
+                    .hasSeverity(Severity.WARNING_NORMAL)
+                    .hasModuleName("module_name_no_package")
+                    .hasPackageName(null);
```

Comment:
This specific test fails and I have absolutely no idea why. If anyone have ideas, I'm all ear :ear: 
I'm not a Java developer, so I might be missing something obvious...

Reply:
`null` is not possible. Seems that the parser returns the module name.

Reply:
`null` is not allowed, use `"-"`